gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -1,49 +1,58 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           meson 1.0
 
 # https://bugzilla.gnome.org/show_bug.cgi?id=636134
 PortGroup           muniversal 1.0
-PortGroup           gobject_introspection 1.0
-PortGroup           active_variants 1.1
 
 name                gstreamer1-gst-plugins-bad
 set my_name         gst-plugins-bad
 # please only commit stable updates (even numbered releases)
-version             1.16.2
-revision            14
+version             1.22.9
+revision            0
 description         A set of plug-ins for GStreamer that need more quality.
 long_description    GStreamer Bad Plug-ins is a set of plug-ins that aren't up to par compared \
                     to the rest. They might be close to being good quality, but they're missing \
-                    something - be it a good code review, some documentation, a set of tests, a \
-                    real live maintainer, or some actual wide use.
-license             GPL LGPL
+                    something - be it a good code review, some documentation, a set of tests, \
+                    a real live maintainer, or some actual wide use.
+license             {GPL LGPL}
```

Comment:
there is a choice between the two licenses or do both apply?

Reply:
Thank you for pointing this out. Let me revert this. (Sorry, missed this part when porting from the earlier version.)